src/modality-specific-files/magnetic-resonance-imaging-data.md

```diff
@@ -165,16 +165,7 @@ whenever possible. See also
 
 ## Anatomy imaging data
 
-<!--
-This block generates a filename templates.
-The inputs for this macro can be found in the directory
-  src/schema/rules/files/raw
-and a guide for using macros can be found at
- https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
--->
-{{ MACROS___make_filename_template("raw", datatypes=["anat"]) }}
-
-Currently supported non-parametric structural MR images include:
+Currently supported non-parametric structural MR images include the following:
```

Comment:
Maybe there could be a little intro to anatomical MRI here?

```suggestion
Anatomy MRI sequences measure static, structural features of the brain.
Each type of contrast has an associated suffix, resulting in many suffixes for this datatype.
For the sake of convenience, these suffixes are broadly divided into two groups: non-parametric and parametric.

Non-parametric suffixes have an arbitrary scale.
For example, T1w data are T1-weighted, but the values do not correspond to actual T1 value estimates.

Parametric suffixes, on the other hand, have a non-arbitrary scale.
A T1map file contains T1 value estimates, in seconds.

### Non-parametric structural MR images

Currently supported non-parametric structural MR images include the following:
```

Reply:
This is a braindump. Definitely don't accept the suggestion as-is!

Reply:
accepting this as is for now

will rework locally